code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -298,31 +300,41 @@
 		if(ignoreslow)
 			return . // Only malusses after here
 
-		var/health_deficiency = max(H.maxHealth - H.health, H.staminaloss)
-		var/hungry = (500 - H.nutrition)/5 // So overeat would be 100 and default level would be 80
-		if(H.reagents)
-			for(var/datum/reagent/R in H.reagents.reagent_list)
-				if(R.shock_reduction)
-					health_deficiency -= R.shock_reduction
-		if(!HAS_TRAIT(H, TRAIT_IGNOREDAMAGESLOWDOWN))
-			if(health_deficiency >= 40)
-				if(flight)
-					. += (health_deficiency / 75)
-				else
-					. += (health_deficiency / 25)
 		if(H.dna.species.spec_movement_delay()) //Species overrides for slowdown due to feet/legs
 			. += 2 * H.stance_damage //damaged/missing feet or legs is slow
 
+		var/hungry = (500 - H.nutrition) / 5 // So overeat would be 100 and default level would be 80
 		if((hungry >= 70) && !flight)
 			. += hungry/50
 		if(HAS_TRAIT(H, TRAIT_FAT))
 			. += (1.5 - flight)
+
 		if(H.bodytemperature < BODYTEMP_COLD_DAMAGE_LIMIT && !(HAS_TRAIT(H, TRAIT_RESISTCOLD)))
 			. += (BODYTEMP_COLD_DAMAGE_LIMIT - H.bodytemperature) / COLD_SLOWDOWN_FACTOR
 
+		var/leftover = .
+		. = round_down(. * SLOWDOWN_MULTIPLIER) / SLOWDOWN_MULTIPLIER //This allows us to round in values of 0.5 A slowdown of 0.55 becomes 1.10, which is rounded to 1, then reduced to 0.5
+		leftover = leftover - .
+
+		var/health_deficiency = max(H.maxHealth - H.health, H.staminaloss)
+		if(H.reagents)
+			for(var/datum/reagent/R in H.reagents.reagent_list)
+				if(R.shock_reduction)
+					health_deficiency -= R.shock_reduction
+		if(!HAS_TRAIT(H, TRAIT_IGNOREDAMAGESLOWDOWN))
+			if(health_deficiency >= 40 - 40 * leftover * SLOWDOWN_MULTIPLIER) //If we have 0.25 slowdown, or halfway to the threshold of 0.5, we reduce the health threshold by that 50%
```

Comment:
the `40 - 40` is confusing here, can you use parentheses to separate out this math operation to make it more readable?
i.e. is it supposed to be `40 - (40 * leftover * SLOWDOWN_MULTIPLIER)` ?

Reply:
Done